Directory.Build.props

```diff
@@ -171,7 +178,8 @@
       linux-musl-arm64;
       linux-x64;
       linux-arm;
-      linux-arm64
+      linux-arm64;
+      freebsd-x64
```

Comment:
@MichaelSimons Can you provide the background why this was added?

Reply:
FreeBSD is a community effort which we want to foster.  
Issue - https://github.com/dotnet/source-build/issues/1139
Initial PR - https://github.com/dotnet/source-build/pull/1362